Resources/Prototypes/Loadouts/Jobs/Medical/medical_doctor.yml

```diff
@@ -2,23 +2,11 @@
 - type: loadoutEffectGroup
   id: SeniorPhysician
   effects:
-  - !type:JobRequirementLoadoutEffect
```

Comment:
If these are Wizden requirements then this needs to be commented out instead of being deleted, and needs Delta-V comments